azure-pipelines/VSMarketplaceTools.ps1

```diff
@@ -6,25 +6,26 @@ $vsixUploadEndpoint = "https://www.vsixgallery.com/api/upload"
 #$vsixUploadEndpoint = "https://localhost:44372/api/upload"
```

Comment:
@AArnott This file is for upload to https://www.vsixgallery.com/.  There is https://marketplace.visualstudio.com/ called VS marketplace. Looks like they are different galleries, right?

Reply:
I've never heard of vsixgallery.com. And notably absent from this script is any mention of a 'secret' or token or anything that would typically be required to upload a VSIX someplace.

Reply:
> I've never heard of vsixgallery.com. And notably absent from this script is any mention of a 'secret' or token or anything that would typically be required to upload a VSIX someplace

I happen to know vsixgallery.com by working on this PR. :) Yes, I would expect some kind of 'secret' is required, but apparently, not for vsixgallery.com.